src/pages/TaskForm.js

```diff
@@ -39,65 +38,66 @@ export default function TaskForm({ pets, addTask }) {
-              placeholder="Insert description"
-              onChange={(e) => setDescription(e.target.value)}
-              required
-              autoFocus
-            />
-          </label>
-        </div>
-        <div className="date">
-          <label>
-            Date*
-            <input
-              type="date"
-              value={date}
-              onChange={(e) => setDate(e.target.value)}
-              required
-            />
-          </label>
-        </div>
-        <div className="time">
-          <label>
-            Time*
-            <input
-              type="time"
-              value={time}
-              onChange={(e) => setTime(e.target.value)}
-              required
-            />
-          </label>
-        </div>
-        <div className="person">
-          <label>
-            To be completed by*
-            <input
-              type="text"
-              value={person}
-              maxLength="100"
-              placeholder="Insert person to complete task"
-              onChange={(e) => setPerson(e.target.value)}
-              required
-            />
-          </label>
-        </div>
-        <SubmitButton text="Submit" />
-        <p>*Mandatory Fields</p>
-      </Form>
+      <main>
+        <Form onSubmit={handleSubmit}>
+          <div className="cancel">
+            <Link to={`/pet/${pet.id}`}>
+              <CancelButton />
+            </Link>
+          </div>
+          <div className="description">
+            <label>
+              Description*
+              <input
+                type="text"
+                value={description}
+                maxLength="100"
+                placeholder="Insert description"
+                onChange={(e) => setDescription(e.target.value)}
+                required
+                autoFocus
+              />
+            </label>
+          </div>
+          <div className="date">
+            <label>
+              Date*
+              <input
+                type="date"
+                value={date}
+                onChange={(e) => setDate(e.target.value)}
+                required
+              />
+            </label>
+          </div>
+          <div className="time">
+            <label>
+              Time*
+              <input
+                type="time"
+                value={time}
+                onChange={(e) => setTime(e.target.value)}
+                required
+              />
+            </label>
+          </div>
+          <div className="person">
+            <label>
+              To be completed by*
+              <input
+                type="text"
+                value={person}
+                maxLength="100"
+                placeholder="Insert person to complete task"
+                onChange={(e) => setPerson(e.target.value)}
+                required
+              />
+            </label>
+          </div>
+          <SubmitButton text="Submit" />
+          <p>*Mandatory Fields</p>
```

Comment:
Are the  * typos or do they be intended?

Reply:
they are intended :)